components/automate-cli/cmd/chef-automate/verify_test.go

```diff
@@ -145,6 +146,82 @@ func TestRunVerifyCmd(t *testing.T) {
 
 }
 
+func TestGetHostIPsWithNoLatestCLI(t *testing.T) {
```

Comment:
what if some ips are reachable and some are not reachable - does it get the version from the reachable ones ?
Write test case for some nodes with existing service and some without

Reply:
Yes. The function will get the IPs of unreachable nodes too. Along with the IPs where the services are not running.

Reply:
Done